src/components/inquiry_artwork_brick/details.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+import colors from "../../assets/colors"
+import TextLink from "../text_link"
+
+export interface DetailsProps extends React.HTMLProps<ArtworkDetails> {
+  artwork: any
```

Comment:
I was missing some of the properties on the artist object. I figured this out and realized I needed to add them to the `RelayProps` interface. However, now it doesn't like the types passed to the `reduce` method called on the artists collection.